src/reference_model/poseidon_python/basic.py

```diff
@@ -0,0 +1,106 @@
+import math
+import os
+
+# Poseidon Constants Parameters
+
+P = 0x73EDA753299D7D483339D80809A1D80553BDA402FFFE5BFEFFFFFFFF00000001
+M = 128
+t_range = {3, 5, 9, 12}
+s_box_exp = 5
+
+roundfull = 8
+roundpartial = {3: 55, 5: 56, 9: 57, 12: 57}

```

Comment:
use all capital words for constant naming

Reply:
constants have been named by capital words